pkg/objectstate/sched/sched.go

```diff
@@ -0,0 +1,50 @@
+package sched
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	nrsv1alpha1 "github.com/openshift-kni/numaresources-operator/api/numaresourcesoperator/v1alpha1"
+)
+
+const DefaultSchedulerPluginImage = "quay.io/openshift-kni/scheduler-plugins:4.10-snapshot"
```

Comment:
there is not a safe fallback to be hardcoded; better to have this in the example CR rather than hardcoded; additionally, the `SchedulerImage` field should be mandatory.

Reply:
Yes I noticed that. didn't get the chance to fix yet.